README.md

```diff
@@ -226,166 +121,9 @@ We use the [Development Project Board](https://github.com/orgs/getAlby/projects/
 - Have a look at this Readme. Can it be improved? Do you see typos? You can open a PR or reach out to us in [our community chat](https://bitcoindesign.slack.com/archives/C02591ADXM2).
 - You can help with [translations](#translations)
 
```

Comment:
same as above, repetition in contribution.md

Reply:
It is indeed repeated. If i put these in the developer document, others will not see them
So I still don't know how to deal with it

I think the contributions should be divided into these aspects

1. Developers and designers contribute to the alby extension, which is related to the contribution documents
2. Developers use alby to develop lightning network dapp, which can be regarded as alby's ecological contribution, such as in html, vue, react and next How to use alby to complete login, authorization, transfer and payment in js? They can develop a single component library, such as www.getmash.com. They have many front-end framework plug-ins
3 Record the experience of using alby. These may not be about writing code, but about writing articles and videos. This may be Evangelist, Developer Advocate, or kol

I feel that relevant documents need to be put in

Reply:
Let me modify it first to see the effect